modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-attaching-vm-secondary-network-cli_{context}"]
+= Attaching a virtual machine to a secondary network in the CLI
+
+Attach a virtual machine or virtual machine instance to a secondary network by adding a bridge to the `spec.domain.devices.interfaces` field and specifying the network attachment definition to the `spec.networks` field in the virtual machine configuration.
```

Comment:
`interfaces` and `networks` paths on vms and vmis are different (vm's path is spec.template.<vmi's-path>). Maybe we should focus on vm?

Reply:
Ah okay. I was wondering about that (whether to keep VM and VMI) when I pulled this content out of virt-creating-bridge-nad-cli.adoc
Most of our docs focus on VMs, as in the past VMIs were more hidden in the UI. I think it makes sense to focus on VMs but we can now create VMIs directly (in the UI) without a VM object. Also if we add this NIC to a VM config while it's running, do we have to restart the VM for the change to come into effect?

Reply:
I'm not sure you can change interfaces on a running vmi instance. @phoracek ?

I concur using VMs in the documentation is preferable. IMO having separate explanations for vms, and vmi might make things more complex for the user. Also, Mac allocation via Kubemacpool is only supported if the vmi was created from a vm.

Reply:
There is no hotplug, so we cannot change interfaces on a running VMI.

Reply:
> There is no hotplug, so we cannot change interfaces on a running VMI.
No hotplug, but you can add/remove nics/disks to a VM. It won't take any effect until the VM is restarted though

Reply:
To VM, yes. Not sure about VMI - there is no hard restart there, you can only delete it and recreate, but then it is a new VMI. I may be wrong though.

Reply:
Thanks for the clarification, folks. I've removed instances of VMI here so we only focus on VMs. Have also added a para and optional step about the need to restart the VM if editing a running VM (or to shutdown before editing).